server/client.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
client.go on the server directory? maybe the name is odd ?

Reply:
This struct purpose was to describe a new client on the chat room. I renamed it in order to make it clearer.